styles.css

```diff
@@ -50,7 +57,8 @@ img {
     margin: 0 auto;
 }
 
-.cards {
+.card {
+    border-radius: 5px;
```

Comment:
this border radius still doesn't do anything - either make it make a difference or remove the class from both the css and html